app/models/knight.rb

```diff
@@ -1,15 +1,16 @@
 class Knight < Piece
-  def is_obstructed?(dest_x, dest_y)
+  def is_obstructed?(_dest_x, _dest_y)
```

Comment:
What possible reason could you have for adding these leading underscores to my variables Jeff?